frontstage/controllers/conversation_controller.py

```diff
@@ -157,28 +164,51 @@ def send_message(
             survey_id = None
 
         sm_v2_thread = {
-            "subject": subject,
-            "category": category,
+            "subject": form.subject.data,
+            "category": form.category,
             "ru_ref": business_id,
             "survey_id": survey_id,
-            "respondent_id": party_id,
+            "respondent_id": form.party_id,
             "is_read_by_respondent": False,
             "is_read_by_internal": False,
         }
         sm_v2_thread_json = _post_to_secure_message_v2_threads(session, headers, sm_v2_thread)
 
         sm_v2_message = {
             "thread_id": sm_v2_thread_json["id"],
-            "body": body,
+            "body": form["body"].data,
             "is_from_internal": False,
-            "sent_by": party_id,
+            "sent_by": form.party_id,
         }
 
         sm_v2_message_json = _post_to_secure_message_v2_messages(session, headers, sm_v2_message)
 
     return sm_v2_message_json["id"] if current_app.config["SECURE_MESSAGE_VERSION"] == "v2" else sm_v1_msg_id
 
 
+def secure_message_enrolment_options(party_id: UUID, secure_message_form: SecureMessagingForm) -> dict:
+    """returns a dict of enrolment options based on a respondents party_id"""
+
+    business_options, survey_options = _get_unique_business_survey_options(party_id)
+
+    sm_enrolment_options = {
+        "survey": _create_formatted_option_list(
```

Comment:
If i have multiple surveys for multiple businesses, there is nothing stopping me from sending a message with the wrong survey for that business